convex/engine/game.ts

```diff
@@ -1,18 +1,23 @@
-import { Infer, Validator } from 'convex/values';
 import { Id } from '../_generated/dataModel';
 import { MutationCtx } from '../_generated/server';
 import { FunctionReference, Scheduler } from 'convex/server';
 
-export type InputHandler<Args extends any, ReturnValue extends any> = {
-  args: Validator<Args, false, any>;
-  returnValue: Validator<ReturnValue, false, any>;
-};
+// TODO: If it's useful, we can try to make this type-safe.
+// For now, the only caller of it is typed specifically for the game.
+// export type InputHandler<
+//   Args extends PropertyValidators,
+//   ReturnValue extends any,
+//   G extends any,
+// > = {
+//   args: Args;
+//   handler: (game: G, now: number, args: ObjectType<Args>) => Promise<ReturnValue>;
+// };
 
-export type InputHandlers = Record<string, InputHandler<any, any>>;
+// export type InputHandlers = Record<string, InputHandler<any, any, any>>;
```

Comment:
let's just remove for now then? (and below)

Reply:
sounds good. wanted to keep it around while you checked it out to see if you wanted to try to add something here short-term. we can find it in the git tree if we need to